<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDistanceResultEstimateDlg</class>
 <widget class="QDialog" name="QDistanceResultEstimateDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>876</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLabel" name="m_labName">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="m_labType_2">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>误差类型</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="m_labRel">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>相对误差(%)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="m_labAbs">
     <property name="text">
      <string>绝对误差</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="m_labAbs_2">
     <property name="text">
      <string>绝对误差(-)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="m_lab1">
     <property name="text">
      <string>距离I段动作时间(s)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="CExBaseListComboBox" name="m_cmb1">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QSettingLineEdit" name="m_txtRel1">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="m_labPlus_1">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtPlus_1">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="m_labMinus_1">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtMinus_1">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="m_lab2">
     <property name="text">
      <string>距离Ⅱ段动作时间(s)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="CExBaseListComboBox" name="m_cmb2">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSettingLineEdit" name="m_txtRel2">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="m_labPlus_2">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtPlus_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="m_labMinus_2">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtMinus_2">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="m_lab3">
     <property name="text">
      <string>距离Ⅲ段动作时间(s)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="CExBaseListComboBox" name="m_cmb3">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSettingLineEdit" name="m_txtRel3">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="m_labPlus_3">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtPlus_3">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="m_labMinus_3">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtMinus_3">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="m_lab4">
     <property name="text">
      <string>距离IV段动作时间(s)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="CExBaseListComboBox" name="m_cmb4">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSettingLineEdit" name="m_txtRel4">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="m_labPlus_4">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtPlus_4">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="m_labMinus_4">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtMinus_4">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="6">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="m_lab5">
     <property name="text">
      <string>距离V段动作时间(s)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="CExBaseListComboBox" name="m_cmb5">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSettingLineEdit" name="m_txtRel5">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="m_labPlus_5">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtPlus_5">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="5">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="m_labMinus_5">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSettingLineEdit" name="m_txtMinus_5">
       <property name="minimumSize">
        <size>
         <width>170</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="1" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnOK">
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_btnCancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSettingLineEdit</class>
   <extends>QLineEdit</extends>
   <header>../../UI/Controls/SettingCtrls/qsettinglineedit.h</header>
  </customwidget>
  <customwidget>
   <class>CExBaseListComboBox</class>
   <extends>QComboBox</extends>
   <header>../../../../Module/BaseClass/QT/QExBaseListComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
